### DISCUSSION TITLE
How to prevent WAF to block Next Image Requests?

### DISCUSSION THREAD

**Discussion by jmacbhc** [vercel/next.js#69149] [Help]
### Summary

I recently deployed my application to production, but I encountered an issue where image requests are being blocked by the Web Application Firewall (WAF) due to the image optimization done by the Next Image component. Is there a way to prevent these requests from being blocked while still maintaining the image optimization?

Thanks in advance!

**Comment by wwwdepot** [comment]
Go to WAF vendor website and ask for help there.
